descartes_light/solvers/include/descartes_light/solvers/bgl/dfs_add_all_solver.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2021, Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef DESCARTES_LIGHT_DFS_ALL_SOLVER_H
```

Comment:
I would rename this file to something like `bgl_dfs_solver` since it is anticipated that more than just the DFS "add-all" solver will be implemented

Reply:
Also our standard is to make the header guard match the file directory -> `DESCARTES_LIGHT_SOLVERS_BGL_DFS_ADD_ALL_SOLVER_H`